src/include/duckdb/execution/operator/join/physical_hash_join.hpp

```diff
@@ -34,6 +72,9 @@ class PhysicalHashJoin : public PhysicalComparisonJoin {
 	vector<LogicalType> build_types;
 	//! Duplicate eliminated types; only used for delim_joins (i.e. correlated subqueries)
 	vector<LogicalType> delim_types;
+	//! Checks and execute perfect hash join optimization
+	unique_ptr<PerfectHashJoinExecutor> pjoin_executor {nullptr};
```

Comment:
nit: this initializer isn't required: `unique_ptr` is always initialized to nullptr.